README.md

```diff
@@ -1 +1,62 @@
-# microverse-week-1
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> Set up the project, the goal here is to master all of the tools and best practices about Linter and Gitflow.
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@hendridg](https://github.com/hendridg)
+- Twitter: [@hendridg](https://twitter.com/hendridg)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/hendridg)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] All of these sections should be valid, I mean they should contain your own information.

Reply:
Thank you @andersonlebon, you really help me a lot